spec/abstractops.html

```diff
@@ -731,6 +731,22 @@ <h1>ISO 8601 grammar</h1>
           ASCIISign
           U+2212
 
+      UnpaddedHour :
+          DecimalDigit
+          `11`
+          `12`
+          `13`
+          `14`
+          `15`
+          `16`
+          `17`
+          `18`
+          `19`
```

Comment:
This list still omits `10`

To reduce the chance of errors like that, consider writing it like this:
```suggestion
          `1` DecimalDigit
```